vpr/src/route/connection_box_lookahead_map.cpp

```diff
@@ -110,16 +110,7 @@ static vtr::Point<T> closest_point_in_rect(const vtr::Rect<T>& r, const vtr::Poi
 }
 
 // resize internal data structures
```

Comment:
This comment is old, needs to be updated?

Reply:
Fixed, thanks.